roles/etcd/meta/main.yml

```diff
@@ -16,6 +16,5 @@ galaxy_info:
   - cloud
   - system
 dependencies:
-- { role: openshift_docker }
```

Comment:
That dependency was already there so we should probably double check things with this removed.